policyengine_us/variables/gov/states/ar/tax/income/deductions/itemized/ar_itemized_deductions.py

```diff
@@ -0,0 +1,53 @@
+from policyengine_us.model_api import *
+
+
+class ar_itemized_deductions(Variable):
+    value_type = float
+    entity = TaxUnit
+    label = "Arizona itemized deductions"
+    unit = USD
+    definition_period = YEAR
+    reference = "https://www.dfa.arkansas.gov/images/uploads/incomeTaxOffice/2022_AR1000F_and_AR1000NR_Instructions.pdf#page=21"
+    defined_for = StateCode.AR
+
+    def formula(tax_unit, period, parameters):
+        person = tax_unit.members
+        p = parameters(period).gov.irs.deductions.itemized
+        less_salt_deds = tax_unit("itemized_deductions_less_salt", period)
+
+        agi = tax_unit("adjusted_gross_income", period)
```

Comment:
the medical expense deduction is capped at the federal 10% as opposed to 7.5% https://law.justia.com/codes/arkansas/2018/title-26/subtitle-5/chapter-51/subchapter-4/section-26-51-423/

- make a parameter for 10% and follow the form:
![image](https://github.com/PolicyEngine/policyengine-us/assets/110687043/4282fc9a-7fde-46ee-aecd-1010b76c7d29)